### PR TITLE
Add schedule

### LINKED ISSUE #33: Add schedule

### PR DESCRIPTION
**Pull request by erfanio**
- Up next on the home screen
- Schedule page

closes #33

### CODE REVIEW

File: src/components/ScheduleItem/index.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.schedule__title {
+  display: flex;
+  flow-flow: row;
```
Comment (josh-) [inline src/components/ScheduleItem/index.scss]: Is this supposed to be `flex-flow`?

File: src/constants.js [hunk 2]
```diff
@@ -1,2 +1,5 @@
 // eslint-disable-next-line import/prefer-default-export
 export const BASE_URL = 'http://localhost:3000/api/v1';
+export const NOTIFICATION_TYPE_TECH_TALK = 'techTalk';
+export const NOTIFICATION_TYPE_SESSION = 'session';
+export const NOTIFICATION_TYPE_SPECIAL = 'mealsRafflesEtc';
```
Comment (josh-) [inline src/constants.js]: "Etc" 😂

File: src/components/ScheduleItem/index.scss (hunk 1 above)
Comment (erfanio) [reply src/components/ScheduleItem/index.scss]: ah god... we need linter for css

Comment (josh-) [reply src/components/ScheduleItem/index.scss]: brb making a npm module which lints people's linters

File: src/constants.js (hunk 2 above)
Comment (erfanio) [reply src/constants.js]: :sweat_smile: courtesy of @dsykesturner https://github.com/unihackhq/skilled-acolyte-ios/blob/5fa8f50d1389f7d7c993acf55b3c279e92ab56ca/Skilled%20Acolyte/ViewControllers/Models/ScheduleItem.swift#L42

Comment (dsykesturner) [reply src/constants.js]: HAha hmmm. Looking over the real schedule I think we could really use an "event" type as well. E.g. Registration Begins, Hacking Ends

Comment (erfanio) [reply src/constants.js]: Maybe we can have a schedule type of `importantMessages` too. That way manual notificaitons and begin/end notification are treated the same

Comment (erfanio) [reply src/constants.js]: or do you think it's worth having a dedicated type for it

Comment (dsykesturner) [reply src/constants.js]: Yeah that could be useful. We need to break up:
- registration begins
- careers fair
- hacking begins/ends
- doors lock
- doors reopen
- presentation stage 1 and 2
- winners announced stage 1 and 2
- closing ceremony

`importantMessages` as well as `event` I recon

Comment (erfanio) [reply src/constants.js]: ah yeh that's quite a few

**Pull request merged**